mne_connectivity/spectral/epochs_multivariate.py

```diff
@@ -296,20 +307,79 @@ def _compute_e(self, csd, n_seeds):
                 "using full rank data or specify an appropriate rank for the "
                 "seeds and targets that is less than or equal to their ranks"
             )
-        T = np.real(T)  # make T real if check passes
+
+        return np.real(T)  # make T real if check passes
+
+    def reshape_results(self):
+        """Remove time dimension from results, if necessary."""
+        if self.n_times == 0:
+            self.con_scores = self.con_scores[..., 0]
+            if self.patterns is not None:
+                self.patterns = self.patterns[..., 0]
+
+
+def _invsqrtm(C, T, n_seeds):
+    """Compute inverse sqrt of CSD over times (used for CaCoh, MIC, & MIM).
+
+    Kept as a standalone function to allow for parallelisation over CSD
+    frequencies.
+
+    See Eq. 3 of Ewald et al. (2012). NeuroImage. DOI:
+    10.1016/j.neuroimage.2011.11.084
+    """
```

Comment:
Even tho it's a private function, can we add a docstring comment here to describe what `C` and `T` are?

Reply:
@orabe can you add this to the docstring when you have time?

Reply:
Done

https://github.com/mne-tools/mne-connectivity/blob/45458e94b54f16e18914ca8ba1a7e692b6989220/mne_connectivity/spectral/epochs_multivariate.py#L324-L349